metricbeat/module/beats/stats/data.go

```diff
@@ -0,0 +1,88 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package stats
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/common"
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstriface"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/beats"
+)
+
+var (
+	schema = s.Schema{
+		"uptime": c.Dict("beat.info.uptime", s.Schema{
+			"ms": c.Int("ms"),
+		}),
+		"runtime": c.Dict("beat.runtime", s.Schema{
+			"goroutines": c.Int("goroutines"),
+		}),
+		"libbeat": c.Dict("libbeat", s.Schema{
+			"output": c.Dict("output", s.Schema{
+				"type": c.Str("type"),
+				"events": c.Dict("events", s.Schema{
+					"acked":      c.Int("acked"),
+					"active":     c.Int("active"),
+					"batches":    c.Int("batches"),
+					"dropped":    c.Int("dropped"),
+					"duplicates": c.Int("duplicates"),
+					"failed":     c.Int("failed"),
+					"toomany":    c.Int("toomany"),
```

Comment:
This `toomany` without underscore looks ugly for me :smile: I suggest to insert an underscore :slightly_smiling_face:

Reply:
I think this comes down to personal preference :smile. I left this as-is so there's a 1-1 correspondence between the API response fields and the event fields.